docker/Dockerfile.linux.amd

```diff
@@ -0,0 +1,10 @@
+FROM alpine:3.6 as alpine
+RUN apk add -U --no-cache ca-certificates
+
+FROM alpine:3.6
+ENV GODEBUG netdns=go
+
+COPY --from=alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
do we need this?

Reply:
I review docker files to match other GA'ed plugins